api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -159,4 +168,28 @@ private void validateStopDatetime(Visit visit, Visit otherVisit, Errors errors)
 		}
 		
 	}
+	
+	private void validateBirthDate(Visit visit, Errors errors) {
+		
+		if (visit.getPatient().getBirthdate() != null && visit.getStartDatetime() != null) {
+			
+			Date birthday = visit.getPatient().getBirthdate();
```

Comment:
can you please extract the logic of getting/calculating the patient birthdate into a private method

`getPatientBirthdateAdjustedIfEstimated`